README.md

```diff
@@ -333,7 +333,7 @@ For example:
 
 ### ⭐️ Keeping existing files `keep_files`
 
-By default, existing files in the publish branch are removed before adding the ones from publish dir. If you want the action to add new files but leave existing ones untouched, set the optional parameter `keep_files` to `true`.
+By default, existing files in the publishing directory (or `destination_dir` if you use) will be removed. If you want the action to add new files but leave existing ones untouched, set the optional parameter `keep_files` to `true`.
```

Comment:
```suggestion
By default, existing files in the publish branch (or just in `destination_dir` if given) will be removed. If you want the action to add new files but leave existing ones untouched, set the optional parameter `keep_files` to `true`.
```

Publish branch was correct, I think. It's how the option is named, but it just did not mention destination dir yet. Thus my suggestion above.